marconi-chain-index/src/Marconi/ChainIndex/Api/JsonRpc/Endpoint/EpochState.hs

```diff
@@ -23,9 +30,49 @@ import Network.JsonRpc.Types (JsonRpc, JsonRpcErr)
 type RpcEpochActiveStakePoolDelegationMethod =
   JsonRpc
     "getActiveStakePoolDelegationByEpoch"
-    EpochState.ActiveSDDByEpochNoQuery
+    Word64
     String
-    (Core.Result EpochState.ActiveSDDByEpochNoQuery)
+    [ActiveSDDResult]
+
+data ActiveSDDResult
+  = ActiveSDDResult
+      !C.PoolId
+      !C.Lovelace
+      !C.SlotNo
+      !(C.Hash C.BlockHeader)
```

Comment:
@berewt : change this to Maybe, and also in legacy, also make the type a record (just here)